package.json

```diff
@@ -2,6 +2,9 @@
   "name": "bowery-chrome-extension",
   "version": "2.6.1",
   "description": "Chrome extension for Bowery appraisers using the Bowery Authorship Application.",
+  "engines": {
+    "node": ">=16.0"
```

Comment:
Please, consider updating Node version to 18+ here or creating a tech debt ticket for it, so we can pick it up soon.

Reply:
https://bowery.atlassian.net/browse/WEB-10123